cell_tree2d/run_test.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+
+cd $SRC_DIR
+python setup.py test
```

Comment:
OK, now I need to see how you got teset into setup.py -- oddlyl, that's not obvious from the pytest docs....